stable/memcached/templates/NOTES.txt

```diff
@@ -1,3 +1,6 @@
+{{- if not .Values.statefulSet }}
+Warning: Memcached has been deployed as deployment, state may not be preserved across node restarts!
+{{ end }}
```

Comment:
Memcached doesn't have much state, being in-memory and all. The reason for using statefulset is to have more predictable pod names (necessary when you want more memcached servers, not loadbalanced) and more control over restarts. I think people should be well aware of this difference between deployment and statefullset, so no need for this warning.

Reply:
Removed.